tooling-observability/src/main/java/org/jboss/eap/qe/observability/containers/OpenTelemetryCollectorContainer.java

```diff
@@ -103,16 +110,38 @@ private OpenTelemetryCollectorContainer() {
 
     public static synchronized OpenTelemetryCollectorContainer getInstance() {
         if (INSTANCE == null) {
-            jaegerContainer = JaegerContainer.getInstance();
             INSTANCE = new OpenTelemetryCollectorContainer();
-            INSTANCE.start();
         }
         return INSTANCE;
     }
 
+    public static synchronized OpenTelemetryCollectorContainer getInstance(JaegerContainer jaegerBackendContainer) {
```

Comment:
Can you please add javadoc to new public methods?

Reply:
+1, sure! Fixing...